tests/middlewares/redirect_follower_tests.rb

```diff
@@ -30,6 +30,72 @@
   env_restore
 end
 
+Shindo.tests('Excon redirector support with too many redirects') do
+  env_init
+
+  tests("request(:method => :get, :path => '/old').body").raises(Excon::Errors::TooManyRedirects) do
```

Comment:
On further consideration, I think we can omit this test to keep things simple, the redirect loop should be good enough I think. Does that make sense?